client/src/components/events/trending_events.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useEffect, useState } from 'react'
```

Comment:
please refer to the comments made on normal_events.tsx and apply similar changes to this file!